projects/components/src/splitter/splitter.component.scss

```diff
@@ -0,0 +1,81 @@
+@import 'mixins';
+
+:host {
+  width: 100%;
+  height: 100%;
+}
+
+.splitter-container {
+  display: flex;
+  width: 100%;
+  height: 100%;
+
+  &.horizontal {
+    flex-direction: row;
+  }
+
+  &.vertical {
+    flex-direction: column;
+  }
+
+  .splitter-content {
+    display: flex;
+  }
+
+  .splitter-content > p {
+    width: 100%;
+    height: 100%;
+  }
+
+  .splitter {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    /* Prevent the browser's built-in drag from interfering */
+    user-select: none;
+    opacity: 0;
+    border-radius: 4px;
+
+    &:hover {
+      background-color: $gray-2;
+      opacity: 1;
+    }
+
+    .cursor {
+      width: 100%;
+      height: 100%;
+    }
+
+    &.horizontal {
+      height: 100%;
+
+      .cursor {
+        width: 100%;
+        height: 50px;
+      }
+
+      &:hover {
+        cursor: col-resize;
+      }
+    }
+
+    &.vertical {
+      width: calc(100% - 72px);
```

Comment:
shouldn't require calc IMO, should find a different way instead

Reply:
calc() offsets are often a sign that a flexbox is using 100% height incorrectly.